security/rbac/delta_configs/ksql-server.properties.delta

```diff
@@ -19,12 +19,9 @@ public.key.path=/tmp/tokenPublicKey.pem
 confluent.metadata.bootstrap.server.urls=http://localhost:8090
 confluent.metadata.http.auth.credentials.provider=BASIC
 confluent.metadata.basic.auth.credentials.provider=USER_INFO
-confluent.metadata.basic.auth.user.info=ksql:ksql1
-http.auth.credentials.provider=BASIC
-basic.auth.credentials.provider=USER_INFO
-basic.auth.user.info=ksql:synt@x3rror
+confluent.metadata.basic.auth.user.info=ksqlserver:ksqlserver1
 
 # Credentials for Schema Registry access
 ksql.schema.registry.url=http://localhost:8081
 ksql.schema.registry.basic.auth.credentials.source=USER_INFO
```

Comment:
@spena -- I validated there are two places this can be removed, see https://github.com/confluentinc/examples/pull/204/commits/0805bad2de30f70f9bef98830c2ffbd54765828d

(cc: @JimGalasyn , @stan-confluent )